testing/test_case.go

```diff
@@ -0,0 +1,4 @@
+package testing
+
+type TestCase struct {
```

Comment:
We can add some helper methods in the future, such as: `Get`, `Post`, etc., like Larave.